packages/mui-material/src/styles/ThemeProvider.test.tsx

```diff
@@ -95,5 +95,18 @@ describe('ThemeProvider', () => {
 
       expect(getByTestId('mode-switcher')).to.have.property('value', 'dark');
     });
+
+    it('should be dark by default', () => {
```

Comment:
Reading this test title without taking a look at the test implementation makes me think the default mode in `ThemeProvider` should be dark. Maybe this is better?

```suggestion
    it('allows default mode to be changed', () => {
```

Reply:
better.